report_template.tex

```diff
@@ -134,6 +134,40 @@ \section{Results and Discussion}
   \label{fig:samplefigure}
 \end{figure}
 
+If you have multiple figures that need grouping together, then you may use figures like Figure~\ref{fig:subcaptionfigure}, and each subfigure can be referenced as Figure~\ref{fig:1b}. It is important to be aware that using subfigures inside \LaTeX will alter the scaling of the image, including the fontsizes, and therefore this should be accounted for while producing the graphs.
+
+\begin{figure}[H]
+	\begin{subfigure}[b]{.5\textwidth}
+		\centering\includegraphics[width=\textwidth]{modelfigure_1p0.pdf}
+		\caption{Noise factor = 1.0}\label{fig:1a}
+	\end{subfigure}\hfill
+	\begin{subfigure}[b]{.5\textwidth}
+		\centering\includegraphics[width=\textwidth]{differentialfigure_1p0.pdf}
+		\caption{Noise factor = 1.0}\label{fig:1b}
+	\end{subfigure}\hfill
+	\begin{subfigure}[b]{.5\textwidth}
+		\centering\includegraphics[width=\textwidth]{modelfigure_10.pdf}
+		\caption{Noise factor = 10}\label{fig:1c}
+	\end{subfigure}\hfill
+	\begin{subfigure}[b]{.5\textwidth}
+		\centering\includegraphics[width=\textwidth]{differentialfigure_10.pdf}
+		\caption{Noise factor = 10}\label{fig:1d}
+	\end{subfigure}\hfill
+	\begin{subfigure}[b]{.5\textwidth}
+		\centering\includegraphics[width=\textwidth]{modelfigure_100.pdf}
+		\caption{Noise factor = 100}\label{fig:1e}
+	\end{subfigure}\hfill
+	\begin{subfigure}[b]{.5\textwidth}
+		\centering\includegraphics[width=\textwidth]{differentialfigure_100.pdf}
+		\caption{Noise factor = 100}\label{fig:1f}
+	\end{subfigure}
+	\caption[Short caption which will be in the table of figures]{Example of a figure that contains multiple subfigures.  For more options on using subfigures and more complicated subcaptions, see the package documentation at \url{https://gitlab.com/axelsommerfeldt/caption} To update this figure, run samplefigure.py and recompile the document.}
```

Comment:
Doesn't this clash with the other caption?

Reply:
Specifically the short caption in square brackets.

Reply:
I kept it as a placeholder for template purposes, but I can make it unique without a problem.